pisa/stages/utils/pi_fix_error.py

```diff
@@ -0,0 +1,78 @@
+"""
+Stage to take the initial errors of MC and keep them
+for all minimization.
+
+Needed to allow mod_chi2 to behave correctly
+"""
+
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+
+from pisa import FTYPE
+from pisa.core.pi_stage import PiStage
+from pisa.utils.profiler import profile
+from pisa.utils import vectorizer
+
+
+class pi_fix_error(PiStage):  # pylint: disable=invalid-name
+    """
+    stage to histogram events
```

Comment:
change docstring accordingly

Reply:
Sorry.  Changed.  Not sure how descriptive to make it....